src/structures/interfaces/InteractionResponses.js

```diff
@@ -64,13 +63,18 @@ class InteractionResponses {
    *   .catch(console.error);
    * @example
    * // Create an ephemeral reply
-   * interaction.reply('Pong!', { ephemeral: true })
+   * interaction.reply({ content: 'Pong!', ephemeral: true })
    *   .then(console.log)
    *   .catch(console.error);
    */
-  async reply(content, options) {
+  async reply(options) {
     if (this.deferred || this.replied) throw new Error('INTERACTION_ALREADY_REPLIED');
-    const apiMessage = content instanceof APIMessage ? content : APIMessage.create(this, content, options);
+
+    let apiMessage;
+    if (options instanceof APIMessage) apiMessage = options;
+    else if (typeof options === 'string') apiMessage = APIMessage.create(this, options);
+    else apiMessage = APIMessage.create(this, options);
```

Comment:
```suggestion
    if (options instanceof APIMessage) apiMessage = options;
    else apiMessage = APIMessage.create(this, options);
```
The `else if` and `else` seem to do exactly the same, not sure if I'm missing something

Reply:
Yes, thanks, must've also slipped through when redoing it after the initial PR